node/config.go

```diff
@@ -495,6 +496,12 @@ func makeAccountManager(conf *Config) (*accounts.Manager, string, error) {
 				backends = append(backends, trezorhub)
 			}
 		}
+		// Start a smart card hub
+		if schub, err := scwallet.NewHub(scwallet.Scheme, keydir); err != nil {
```

Comment:
Could you add integration into Clef aswell? `signer//core/api.go:StartClefAccountManager`

Reply:
as discussed this morning, this is for a subsequent PR